Carbon.WebApplication.SolutionService/Carbon.WebApplication.SolutionService.csproj

```diff
@@ -1,9 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <Version>2.8.1</Version>
-    <Description>-2.8.0
+	  <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
+	  <Version>2.9.0</Version>
+    <Description>- 2.9.0
```

Comment:
there is an extra dash left before the version number

Reply:
by design :)